person.rb

```diff
@@ -4,7 +4,7 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] You are required to set both the getter and setter for the rental, and the :rental is rather supposed to be with the attribute accessor and not the attribute reader. Kindly make this change. ```attr_accessor :name, :age, :rentals```.